### PR TITLE
Update catalog-executions-ssisdb-database.md

### PR DESCRIPTION
**Pull request by DanielAdeniji**
Added the following attributes operation_type and created_time

### CODE REVIEW

**Comment by PRMerger10** [comment]
@DanielAdeniji : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/integration-services/system-views/catalog-executions-ssisdb-database.md [hunk 1]
```diff
@@ -35,6 +35,8 @@ ms.author: chugu
 |executed_as_sid|**varbinary(85)**|The SID of the user who started the instance of execution.|  
 |executed_as_name|**nvarchar(128)**|The name of the database principal that was used to start the instance of execution.|  
 |use32bitruntime|**bit**|Indicates if the 32-bit runtime is used to run the package on a 64-bit operating system. When the value is `1`, the execution is performed with the 32-bit runtime. When the value is `0`, the execution is performed with the 64-bit runtime.|  
+|operation_type|**smallint**|The type of operation. The operation_type of 200 means create/start execution'|  
+|created_time|**datetimeoffset**|The time when the instance was initiated.  Keep in mind that there is a lag time betwen initiation and actual start time'|
```
Comment (v-andreaco) [inline docs/integration-services/system-views/catalog-executions-ssisdb-database.md:39]:
```suggestion
|operation_type|**smallint**|The type of operation. The operation_type of 200 means create/start execution.|  
|created_time|**datetimeoffset**|The time when the instance was initiated.  Keep in mind that there is a lag time between initiation and actual start time.| 

```

**Comment by PRMerger10** [comment]
@v-andreaco : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
@chugugrace Could you take a look at the proposed update to your article? 

Thanks!

**Comment by ktoliver** [comment]
Hi @rothja - Would you be able to review or reassign this pull request? Thanks.

**Review by chugugrace — APPROVED** [review]

**Comment by chugugrace** [comment]
#sign-off

**Comment by PRMerger5** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**